Patches/Rim-Hivers!/New_Hivers.xml

```diff
@@ -229,39 +233,39 @@
   <!-- ========== Civil ========== -->
```

Comment:
This patch is failing. I fixed the patch recently, so I think the factiondef names are being incorrectly being reverted.